src/Services/ExtensionService.cs

```diff
@@ -204,6 +206,38 @@ public async Task<IEnumerable<IExtensionWrapper>> GetInstalledExtensionsAsync(bo
         }
     }
 
+    private async Task<IEnumerable<string>> GetInstalledWidgetExtensionsAsync()
+    {
+        await _getInstalledExtensionsLock.WaitAsync();
+        try
+        {
+            if (_installedWidgetsPackageFamilyNames.Count == 0)

```

Comment:
This is fine to unblock xbox, but you'll also want to implement events for loaded/unloaded/updated.  Otherwise widget extensions won't show up until Dev Home is restarted, or they may stick around for longer than they should if the extension is uninstalled.

Reply:
We already do this for Dev Home extensions.  You should be able to follow the same model.

Reply:
I think the event handlers for Dev Home extensions actually cover widget extensions, at least as far as we care -- all three (installing, uninstalling, updating) essentially ask if the extension is (or was) a valid Dev Home extension. If it wasn't, ie. if it was just an app that has a widget associated with it (like Your Phone) then it was never in our list and we don't care about putting it in our list or removing it.

Reply:
Oh good call.  I didn't see you added it to OnPackageChange().  Looks good!